docs/docusaurus/docs/guides/validation/how_to_validate_data_by_running_a_checkpoint.md

```diff
@@ -7,11 +7,22 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 import TechnicalTag from '@site/docs/term_tags/_tag.mdx';
 
+import InProgress from '/docs/components/warnings/_in_progress.md'
```

Comment:
this page has been updated, and the update is reverted as part of this PR!! :( Please remove this file from the PR

Reply:
Yikes.  It should be reverted now.